Automattic/jetpack
# Fix price calculation issue my jetpack

## Proposed changes:

* Use Math.round instead of Math.ceil when removing trailing insignificant 1's in price calculations. Math.ceil was making some prices round up a full cent (ex. 9.95000000001 was being rounded up to 9.96)

### Other information:


## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. Go to `/wp-admin/admin.php?page=my-jetpack#/add-stats` and make sure the price is $8.33 instead of what was previously there; $8.34
![image](https://github.com/Automattic/jetpack/assets/65001528/4f111c21-0b8c-4483-90a2-25652c8a0936)
3. Check the other interstitials to make sure the prices are all correct and none of them are off by 1 cent anymore. You can [find the list of interstitials here](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/my-jetpack/_inc/admin.jsx#L52)
